scripts/install

```diff
@@ -130,11 +129,17 @@ ynh_replace_string --match_string="__PEER_PORT__" --replace_string="$peer_port"
 ynh_replace_string --match_string="__PORT__" --replace_string="$port" --target_file=../conf/settings.json
 cp ../conf/settings.json /etc/transmission-daemon/settings.json
 
+cp ../conf/90-transmission.conf /etc/sysctl.d/90-transmission.conf
+if [ ${PACKAGE_CHECK_EXEC:-0} -eq 0 ]; then
+    sysctl --system
```

Comment:
Why not simply using `sysctl --load=/etc/sysctl.d/90-transmission.conf` ?
It would load only the added file.

Reply:
Because I didn't rtfm